### PR TITLE
[SM64] Add back custom include directory + add automatic option

### LINKED ISSUE #463: Custom exports with export texture as png can't have the png filename changed

### PR DESCRIPTION
**Pull request by Lilaa3**
Fixes #463
Old property was hidden post combined export pr, this adds a new prop for it in the combined exporter and upgrades it

### CODE REVIEW

**Review by ThePerfectMario64 — APPROVED** [review]
Tests went good :+1:

**Comment by ThePerfectMario64** [comment]
@thecozies review please

File: fast64_internal/sm64/sm64_objects.py [hunk 1]
```diff
@@ -2156,6 +2157,10 @@ def draw_export_options(self, layout):
             col.prop(self, "export_bhv")
             self.draw_obj_name(layout)
 
+    @property
+    def actor_names(self) -> list:
+        return list(dict.fromkeys(filter(None, [self.obj_name_col, self.obj_name_gfx])).keys())
```
Comment (Dragorn421) [inline fast64_internal/sm64/sm64_objects.py:2162]: I assume the dict keys business is for de-duplication?
Simpler:
```suggestion
        return list(set(filter(None, [self.obj_name_col, self.obj_name_gfx])))
```

File: fast64_internal/sm64/sm64_objects.py [hunk 2]
```diff
@@ -2277,6 +2283,16 @@ def draw_props(self, layout):
...
             if not self.draw_actor_path(info_box):
                 return
+        elif bpy.context.scene.saveTextures:
+            if self.custom_include_directory:
+                info_box.label(text=f'Include directory "{self.custom_include_directory}"')
+            else:
+                actor_names = self.actor_names
+                joined = ",".join(self.actor_names)
+                if len(actor_names) > 1:
+                    joined = f"{{{joined}}}"
```
Comment (Dragorn421) [inline fast64_internal/sm64/sm64_objects.py:2293]: I find this more readable:
```suggestion
                    joined = "{" f"{joined}" "}"
```

File: fast64_internal/sm64/sm64_objects.py (hunk 1 above)
Comment (Lilaa3) [reply fast64_internal/sm64/sm64_objects.py:2162]: This can change order

Comment (Dragorn421) [reply fast64_internal/sm64/sm64_objects.py:2162]: Sadge ok then

**Review by Dragorn421 — APPROVED** [review]

**Pull request merged**